spec/example_app/app/dashboards/country_dashboard.rb

```diff
@@ -0,0 +1,23 @@
+require "administrate/base_dashboard"
+
+class CountryDashboard < Administrate::BaseDashboard
+  ATTRIBUTES = [
```

Comment:
@nickcharlton done! And should i fix that dashboard attributes definition according to Hound?